python/erdos/__init__.py

```diff
@@ -465,17 +484,19 @@ def sigint_handler(sig, frame):
         0, data_addresses, control_addresses, graph_filename
     )
 
-    return NodeHandle(py_node_handle, processes)
+    return NodeHandle(py_node_handle, processes)  # type: ignore
```

Comment:
This is because of the macOS hack above. Ideally, we should be using an `mp.Process` as the `NodeHandle` describes above. However, retrieving the context yields a `ForkProcess` that does not seem to inherit from `mp.Process`.